ui.glade

```diff
@@ -28,6 +28,20 @@
       </row>
     </data>
   </object>
+  <object class="GtkListStore" id="list_wind_scale">
+    <columns>
+      <!-- column-name column_wind_scale -->
+      <column type="gchararray"/>
+    </columns>
+    <data>
+      <row>
+        <col id="0" translatable="yes">Miles(mph)</col>
```

Comment:
Add space after `Miles` and `Kilos`. And also reorder the menu item, put `Kilos` first to be consistent with `Celsius (°C)`. Means metric system ordered first. 
Besides, currently, if I set to `Miles`. It shows `Kilos` in menu and vice versa.